pyrefly/lib/test/suppression.rs

```diff
@@ -62,3 +62,73 @@ testcase!(
 3 + "3" # E:
 "#,
 );
+
+testcase!(
+    test_pyrefly_suppression_typed,
+    r#"
+def foo() -> str:
+  # pyrefly: ignore[bad-return]
+  return 1
+"#,
+);
+
+testcase!(
+    test_pyrefly_suppression_typed_inline,
+    r#"
+def foo() -> str:
+  return 1  # pyrefly: ignore[bad-return]
+"#,
+);
+
+testcase!(
+    test_pyrefly_suppression_typed_wrong_type,
+    r#"
+def foo() -> str:
+  # pyrefly: ignore[bad-assignment]
+  return 1 # E:
```

Comment:
Is it possible to put the error message after the `E:` so that we can verify the right error message is coming up? Also it serves as good documentation about what an error from this would look like.